### PR TITLE
Support all current Timestream datatypes

### LINKED ISSUE #65: Feature Request: Support for Interval types are missing
When trying to execute a query that returns an interval an error is returned instead of data. 

Example use case: Showing time since last update for a set of sensors

Example query: `select measure_name, now() - max(time) as last_update from dbname."tablename" group by measure_name order by measure_name asc`

Expected outcome: Table with two columns of results (as in AWS Timestream Console)

Actual outcome: Table with just one column and a message of unsupported data type: "Unsupported scalar value: INTERVAL_DAY_TO_SECOND". This seems to originate from file: parsers.go line: 60 where it is clear that Interval datatypes are not yet supported. An example of how seems to be present in the AWS Developer guide page 79 (https://docs.aws.amazon.com/timestream/latest/developerguide/timestream.pdf)

### PR DESCRIPTION
**Pull request by tiratatp**
Here's all supported datatypes: https://docs.amazonaws.cn/sdk-for-go/api/service/timestreamquery/#pkg-constants

This will also fix #65.

Screen Shot 2021-02-15 at 6 14 08 PM

### CODE REVIEW

File: pkg/timestream/parsers.go [hunk 1]
```diff
@@ -56,6 +56,30 @@ func getFieldBuilder(t *timestreamquery.Type) (*fieldBuilder, error) {
 				parser:    datumParserInt32,
 			}, nil
 
+		case timestreamquery.ScalarTypeIntervalDayToSecond:
+			return &fieldBuilder{
+				fieldType: data.FieldTypeNullableString,
```
Comment (ryantxu) [inline pkg/timestream/parsers.go:61]: String does not seem like the right choice... maybe float64 for with the unit seconds?

**Comment by ryantxu** [comment]
@tiratatp -- this looks great, thanks!  

Can you also add a test case for it?  Perhaps update the `select-consts` test with your query that touches all types:
https://github.com/grafana/timestream-datasource/blob/master/pkg/timestream/executor_test.go#L71

You will need to remove the `t.Skip("...` to save the output, this will add/modify the saved output and a 'golden' file like:
https://github.com/grafana/timestream-datasource/blob/master/pkg/timestream/testdata/select-consts.golden.txt

File: pkg/timestream/parsers.go (hunk 1 above)
Comment (tiratatp) [reply pkg/timestream/parsers.go:61]: Okay I'll work on this. Is there a way to debug Go code better? Right now if I'm doing something wrong all I got was a 500 from front end.

Comment (ryantxu) [reply pkg/timestream/parsers.go:61]: >  Is there a way to debug Go code better?

I find the easiest is to initiate things from VSCode in the backend directly.  In my working environment either move the value I care about first or comment out the other variables, then set breakpoints and debug tests from:

image


To debug go code where things are kicked off from the frontend is more complicated!  If the above method does not work for you, I can find some example vscode launcher scripts and add them to our repo

Comment (tiratatp) [reply pkg/timestream/parsers.go:61]: Sounds easy enough. Let me try that.

**Review by ryantxu — APPROVED** [review]
LGTM 

thanks @tiratatp

**Pull request merged**